iulianIachimov/src/main/java/dataCollectionTask/Models/Address.java

```diff
@@ -0,0 +1,32 @@
+package dataCollectionTask.Models;
+
+public class Address {
+  public String country;
+  public String city;
+  public String street;
+
+  public Address(String country, String city, String street) {
+    this.country = country;
+    this.city = city;
+    this.street = street;
+  }
+
+  public String getFullAddress() {
+    return "Full address is " + this.country + " " + this.city + " " + this.street;
```

Comment:
you already have overrided `toString()`

Reply:
The method getFullAddress is not used? Do I need to make any changes here?

Reply:
Sorry, there should be no '?' after first sentence in my last comment. It is an affirmation not question. I have deleted getFullAddress() leaving toString()